src/amcrest/privacy_mode.py

```diff
@@ -0,0 +1,36 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# vim:sw=4:ts=4:et
+
+from .http import Http
+
+
+# This API is based on an undocumented API of Amcrest Cameras.
+# This API may change without notice
+class PrivacyMode(Http):
+    @property
+    def privacy_set(self, mode: bool) -> str:
```

Comment:
Should there be a `@property` and a `@property.setter` pair here? Or should this just be a normal (non-property) method? I don't think it is possible to pass a value in for `mode` if this is a `@property`.

Reply:
I will be honest: I had no idea what that did (python isn't my main lang). I just googled it and now I get it. Fixing.